docs/assets/css/main.scss

```diff
@@ -1,6 +1,5 @@
 body {
-	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
-		Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+	font-family: Roboto, sans-serif;
```

Comment:
This is just for the documentation of kotti!

Reply:
I don't approve of this, if we're going to implement a font then we either require it or we allow fallbacks

the way to solve font in consistency is to include our own font not to compound the problem worse by making none linux platforms render the wrong thing

Finally Ruboto? really I get Open Sans, might as well pick Ariel

Reply:
> Also a very similar font to Roboto that's on every platform is Arial so you can add that in the fallback list

Solved !